services/frontend/src/utils/models/Profile.test.js

```diff
@@ -0,0 +1,46 @@
+import Profile from "./Profile";
+
+describe("normalizeApiResultsForRedux", () => {
+    const ProfilesList = [
+        new Profile({id: "1"})
+    ];
+
+    const ProfilesMap = {
+        [ProfilesList[0].id]: {...ProfilesList[0]},
+    };
+
+    it("normalizes a list of Profiles to a map of Profiles with just skill IDs", () => {
+        expect(Profile.normalizeApiResultsForRedux(ProfilesList)).toEqual(ProfilesMap);
+    });
+});
+
+describe("filterForCurrentuser", () => {
+    const allProfiles = [
+        new Profile({id: "1", name: "bill", userId: "1"}),
+        new Profile({id: "2", contactEmail: "fred@gmail.com", userId: "2"}),
+        new Profile({id: "3", phone: "9051142315", userId: "3"})
+    ];
+
+    const userIds = [
```

Comment:
What's even the point of this variable? It's not used anywhere; not to generate the test data, and not even in the tests.

Reply:
I was passing in userIds[0], etc, etc. 

Turns out I changed that and don't need it anymore, so it's removed now.